tox.ini

```diff
@@ -8,11 +8,13 @@ setenv =
     PYTHONPATH = {toxinidir}/matrix_client
 deps =
     .[test]
+    .[e2e]
```

Comment:
While Travis now runs e2e tests instead of ignoring them, it has lost the ability to tell us if the tests pass without e2e, ie if they are properly ignored. Maybe something should be done about this?